pyvrp/educate/tests/test_LocalSearch.py

```diff
@@ -61,7 +83,10 @@ def test_local_search_set_get_neighbours(
     seed = 42
     rng = XorShift128(seed=seed)
     pen_manager = PenaltyManager(data.vehicle_capacity)
-    ls = LocalSearch(data, pen_manager, rng)
+
+    prev_neighbours = [[client] for client in range(data.num_clients)]
```

Comment:
This makes me think: shouldn't we validate that client can't be a neighbour of client itself? I think u = v gives undefined behavior in the local search. (I understand that will not be the case here if you prepend the depot afterwards.)

Reply:
I've added some checks for this (depot and the client itself).